solutions/devsprint-matheus-reis-1/RealEstateApp/AppDelegate/SceneDelegate.swift

```diff
@@ -19,7 +19,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         guard let windowScene = (scene as? UIWindowScene) else { return }
 
         self.window = UIWindow(windowScene: windowScene)
-        self.window?.rootViewController = UINavigationController(rootViewController: PropertyListViewController())
+        self.window?.rootViewController = UINavigationController(rootViewController: AccountViewController())
```

Comment:
Você fez essa modificação apenas para teste, correto? Se sim, sugiro retornar ao estado anterior, com a `PropertyListViewController`

```suggestion
        self.window?.rootViewController = UINavigationController(rootViewController: PropertyListViewController())
```

Reply:
fiz, mas foi proposital, a tela de login nao seria a primeira do app?